### PR TITLE
Fix multiple shop id functionality.

### PR DESCRIPTION
**Pull request by gyKa**
Related to #88

### CODE REVIEW

File: Sync/Extractor/AbstractExtractor.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+namespace ONGR\ConnectionsBundle\Sync\Extractor;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\DependencyInjection\Exception\InvalidArgumentException;
+
+/**
+ * Common actions required for all extractors.
+ */
+class AbstractExtractor
```
Comment (trylika) [inline Sync/Extractor/AbstractExtractor.php]: if it's abstract why class is not abstract?

File: Tests/Functional/Sync/Extractor/PassthroughExtractorTest.php [hunk 2]
```diff
@@ -71,37 +82,38 @@ public function testExtractForCreateItem()
...
+            $this->assertEquals($id, $actual->document_id);
+            $this->assertEquals($timestamp, new DateTime($actual->timestamp));
+        }
     }
 
     /**
      * Test if extract is able to add data to the storage for item update action.
      */
-    public function testExtractForUpdateItem()
+    public function ttestExtractForUpdateItem()
```
Comment (GrandLTU) [inline Tests/Functional/Sync/Extractor/PassthroughExtractorTest.php]: Why ttest?

File: Tests/Functional/Sync/Extractor/PassthroughExtractorTest.php [hunk 3]
```diff
@@ -109,37 +121,38 @@ public function testExtractForUpdateItem()
...
+            $this->assertEquals($id, $actual->document_id);
+            $this->assertEquals($timestamp, new DateTime($actual->timestamp));
+        }
     }
 
     /**
      * Test if extract is able to add data to the storage for item delete action.
      */
-    public function testExtractForDeleteItem()
+    public function ttestExtractForDeleteItem()
```
Comment (GrandLTU) [inline Tests/Functional/Sync/Extractor/PassthroughExtractorTest.php]: Also whats the reason behind ttest?

File: Tests/Functional/Sync/Extractor/PassthroughExtractorTest.php (hunk 2 above)
Comment (gyKa) [inline Tests/Functional/Sync/Extractor/PassthroughExtractorTest.php]: Good catch!

**Pull request merged**